lectures/time_series_with_matrices.md

```diff
@@ -366,7 +366,7 @@ $$
 以及
 
 $$
-\Sigma_y = \tilde A (\σ_u^2 I_{T \times T} ) \tilde A^T
+\Sigma_y = \tilde A (\sigma_u^2 I_{T \times T} ) \tilde A^T
```

Comment:
@HumphreyYang should these be unicode sigma?

Reply:
I've just left this as a question.

Reply:
Many thanks @mmcky, I think this is correct under the latex environment: )